_layouts/base.html

```diff
@@ -12,6 +12,10 @@
   {% include project-footer.html %}
 
   {% include site-scripts.html %}
+
+  {% if page.mermaid %}
```

Comment:
I guess you have to set this flag on your page now? (because right now it is not rendering)

Reply:
Yes, I haven't pushed the updated post markdown yet.